src/data/faq_de.json

```diff
@@ -214,6 +214,16 @@
                             "<b>Schritt-für-Schritt-Anleitung:</b>",
                             "Um einen Ausdruck Ihres Digitalen COVID Zertifikates der EU zu erstellen, gehen Sie bitte folgendermaßen vor: <ul><li>Tippen Sie auf den „Zertifikate“-Reiter in der Corona-Warn-App</li><li>Wählen Sie eine Person aus, indem Sie in der Zertifikate-Übersicht auf ein entsprechendes Zertifikat tippen</li><li>Scrollen Sie nach unten und wählen Sie ein Zertifikat dieser Person aus</li><li>In dieser Detailansicht tippen Sie oben rechts auf das Drei-Punkte-Menü (Android) beziehungsweise unten auf „Mehr“ (iOS)</li><li>Tippen Sie auf „Druckversion anzeigen“</li><li>Tippen Sie auf „Weiter“</li><li>Tippen Sie oben rechts auf das Symbol für „Drucken” oder für „Teilen”</li><li>In der folgenden Ansicht können Sie das erstellte PDF-Dokument mit den Standardfunktionen des Betriebssystems drucken, in andere Apps importieren oder auch teilen</li></ul>"
                         ]
+                    },
+                    {
+                        "title": "Wie werden Zertifikats durch Dritte geprüft?",
+                        "anchor": "eu_dcc_check",
+                        "active": true,
+                        "textblock": [
+                            "In Deutschland können Dritte nur mit der CovPassCheck-App verlässlich überprüfen, ob es sich um ein valides Impf-, Genesenen-, oder Testzertifikat handelt. Zusätzlich sollte eine Identitätsprüfung mit Lichtbildausweis stattfinden. Bitte beachten Sie, dass in anderen Ländern gegebenenfalls andere Apps zur Zertifikatsprüfung verwendet werden.",
+                            "Eine reine Sichtprüfung der Zertifikate reicht nicht aus, da beispielsweise manipulierte Screenshots, Apps oder Systemeinstellungen (z.&nbsp;B. geändertes Datum) verwendet werden könnten. Die CovpassCheck-App ist eine sichere Lösung für z.B. Gewerbetreibende und Behörden, mit der digitale COVID-Zertifikate der EU zuverlässig geprüft werden können. Weitere Informationen und Download-Links zur CovPassCheck-App finden Sie hier: <a href='https://www.digitaler-impfnachweis-app.de/covpasscheck-app/' target='_blank' rel='noopener noreferrer'>COVID-Zertifikate der EU direkt per App prüfen</a>.",

```

Comment:
The second z.B. should also be written with `&nbsp;`

Reply:
right, thanks